common/djangoapps/student/views.py

```diff
@@ -2524,9 +2528,9 @@ def validate_password(user, password):
             num_days
         ).format(num=num_days)
 
-    is_password_valid = err_msg is None
+    is_security_violated = err_msg is not None
```

Comment:
I don't like this reverse on the return variable. Maybe we can update the variable to something less verbose, like `is_valid`?